WordPress/Classes/ViewRelated/Comments/FullScreenCommentReplyViewController.swift

```diff
@@ -25,18 +25,29 @@ public class FullScreenCommentReplyViewController: EditCommentViewController, Su
     private(set) var replyButton: UIBarButtonItem!
 
     /// Reply Suggestions
-    ///
     private var siteID: NSNumber?
     private var prominentSuggestionsIds: [NSNumber]?
     private var suggestionsTableView: SuggestionsTableView?
     private var searchText: String?
 
+    private var viewModel: FullScreenCommentReplyViewModelType
+
     // Static margin between the suggestions view and the text cursor position
     private let suggestionViewMargin: CGFloat = 5
 
-    var viewModel: FullScreenCommentReplyViewModelType = FullScreenCommentReplyViewModel()
     public var placeholder = String()
 
+    init(viewModel: FullScreenCommentReplyViewModelType = FullScreenCommentReplyViewModel()) {
+        // nibName from super class
+        let nibName = String(describing: EditCommentViewController.self)
+        self.viewModel = viewModel
+        super.init(nibName: nibName, bundle: nil)
```

Comment:
[Nit]
EditCommentViewController has a static function (`+ (NSString *)nibName;`) that determines the nibName that we can leverage. If we make this public we can do something like this:

```suggestion
        self.viewModel = viewModel
        super.init(nibName: Self.nibName(), bundle: nil)
```

What do you think?

Reply:
Nice catch. I applied your suggestions. Thank you!